src/sentry/runner/commands/tsdb.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import absolute_import
+
+from collections import OrderedDict
+from datetime import datetime, timedelta
+
+import click
+import pytz
+from dateutil.parser import parse
+
+from sentry.runner.decorators import configuration
+from sentry.utils.iterators import chunked
+
+
+class DateTimeParamType(click.ParamType):
```

Comment:
I'd love to make this support more flexible inputs types like `git` generally accepts for `--since` and `--until` at some point and move it into `utils` somewhere.